tools/run_tests/performance/scenario_config.py

```diff
@@ -125,8 +127,8 @@ def _ping_pong_scenario(name, rpc_type,
       'client_type': client_type,
       'security_params': _get_secargs(secure),
       'outstanding_rpcs_per_channel': 1,
-      'client_channels': 1,
-      'async_client_threads': 1,
+      'client_channels': client_channels,
+      'async_client_threads': max(1, async_client_threads),
```

Comment:
This is wrong. When it's an unconstrained case, `async_client_threads` should be 0.